libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -176,6 +176,21 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("OPTIONS", 21, AP_BattMonitor_Params, _options, 0),
 
+    // @Param: CELL_V
+    // @DisplayName: Maximum cell voltage for cell count detection
+    // @Description:
+    // @Units: V
+    // @Increment: 0.01
+    // @User: Advanced
+    AP_GROUPINFO("CELL_V", 22, AP_BattMonitor_Params, _cell_detect_volt, 4.25f),
+
+    // @Param: CELL_CNT
+    // @DisplayName: Overrides cell count autodetection if not 0
+    // @Description:
```

Comment:
it miss description that is why it fails the test

Reply:
apparently it was indeed the length...I dont think having a sentence for the display name is going to fly, its what the  description metadata should be.......Display name should be something like the unabbreviated name, ie Cell Count...and short..otherwise it will not display properly in wiki or GCS

Reply:
It was CELL_V when I tried and it was fine, but your are right BATT9_CELL_DET_V is too long too.
The empty description is detection is an issue on our parser, I have made a pr to correct it